spectrocloud/resource_application_profile.go

```diff
@@ -105,6 +105,13 @@ func resourceApplicationProfile() *schema.Resource {
 							Description: "The name of the specified pack.",
 							Required:    true,
 						},
+						"properties": {
+							Type:     schema.TypeMap,
```

Comment:
```suggestion
							Type:     schema.TypeMap,
							Description: "The key and value pair required for dependents packs."
```

I used the definition @SivaanandM created but when I read it I still don't get a good sense of what this is. Is there a better description we can use?

Reply:
updated